CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ option(ENABLE_COMPILER_WARNINGS "Build with compiler warnings" OFF)
 option(ENABLE_SANITIZER "Use memory sanitizer for Debug build" OFF)
 option(ENABLE_TESTS "Enable Valhalla tests" ON)
 option(ENABLE_WERROR "Convert compiler warnings to errors. Requires ENABLE_COMPILER_WARNINGS=ON to take effect" OFF)
+option(ENABLE_BENCHMARKS "Enable microbenchmarking" OFF)
```

Comment:
is there any reason to defaul this to off. i would hate this code to suffer from bit-rot. to me we should keep compiling this if we want to support/use it. that way anyone who makes an incompatible change feels the pain of the build failing :)

Reply:
Done, and added to CI.